### PR TITLE
Improve + add job names

### PR DESCRIPTION
**Pull request by tas50**
Make sure these match what's happening

Signed-off-by: Tim Smith <tsmith84@gmail.com>

### CODE REVIEW

File: terraform-plan/README.md [hunk 1]
```diff
@@ -30,11 +30,12 @@ on:
...
       - "terraform/main.tf"
 jobs:
-  steps:
-    - uses: actions/checkout@v3
-    - uses: mondoohq/actions/terraform-plan@main
-      env:
-        MONDOO_CONFIG_BASE64: ${{ secrets.MONDOO_SERVICE_ACCOUNT }}
-      with:
-        path: terraform
+  scan-tf:
```
Comment (scottford-io) [inline terraform-plan/README.md:33]:
```suggestion
  terraform-plan-security-scan:
```

File: terraform-hcl/README.md [hunk 2]
```diff
@@ -30,11 +30,12 @@ on:
...
       - "terraform/main.tf"
 jobs:
-  steps:
-    - uses: actions/checkout@v3
-    - uses: mondoohq/actions/terraform-hcl@main
-      env:
-        MONDOO_CONFIG_BASE64: ${{ secrets.MONDOO_SERVICE_ACCOUNT }}
-      with:
-        path: terraform
+  scan-tf:
```
Comment (scottford-io) [inline terraform-hcl/README.md:33]:
```suggestion
  terraform-hcl-security-scan:
```

File: terraform-state/README.md [hunk 3]
```diff
@@ -30,11 +30,12 @@ on:
...
       - "terraform/main.tf"
 jobs:
-  steps:
-    - uses: actions/checkout@v3
-    - uses: mondoohq/actions/terraform-state@main
-      env:
-        MONDOO_CONFIG_BASE64: ${{ secrets.MONDOO_SERVICE_ACCOUNT }}
-      with:
-        path: terraform
+  scan-tf:
```
Comment (scottford-io) [inline terraform-state/README.md:33]:
```suggestion
  terraform-state-security-scan:
```

File: README.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ on:
     paths:
       - "k8s/*.yaml"
```
Comment (scottford-io) [inline README.md:48]:
```suggestion
      - "k8s/manifest.yaml"
```

File: README.md [hunk 5]
```diff
@@ -47,7 +47,7 @@ on:
...
 jobs:
-  install:
+  scan:
```
Comment (scottford-io) [inline README.md:50]:
```suggestion
  k8s-manifest-scan:
```

File: policy/README.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ on:
     paths:
       - "policy/policy.yml"
 jobs:
-  install:
+  upload:
```
Comment (scottford-io) [inline policy/README.md:37]:
```suggestion
  upload-policy-bundle:
```

**Review by scottford-io — CHANGES_REQUESTED** [review]
@tas50 I love this! It was on my list as well, so THANK YOU!!!

I made some suggestions. I think we should provide names that are very descriptive. An example is that if we just use something like `scan-tf` for HCL, plan, and state jobs, then you would end up with three jobs that are named the same.

**Comment by tas50** [comment]
Merging after pulling in the suggestions from Scott

**Pull request merged**